bluebottle/recurring_donations/mails.py

```diff
@@ -19,16 +20,17 @@ def mail_monthly_donation_processed_notification(monthly_order):
 
     subject = _("Thank you for your monthly support")
 
-    translation.activate(cur_language)
+    translation.activate(receiver.primary_language)
 
     send_mail(
-        template_name='monthly_donation.mail',
+        template_name='recurring_donations/mails/monthly_donation.mail',
         subject=subject,
+        site=tenant_url(),
+        to=receiver,
         order=monthly_order,
-        receiver_first_name=receiver.first_name.capitalize(),
+        first_name=receiver.first_name.capitalize(),
```

Comment:
Do we need to change the email templates? We'll lose the translation strings and we need to regenerate and re-enter new translations?